app.py

```
@@ -113,32 +304,56 @@
+
 
         try:
             # Fetch course name
             cursor.execute("SELECT name FROM courses WHERE id = ?", (course_id,))
             course_res = cursor.fetchone()
             if course_res:
                 selected_course_name = course_res['name']
+                print(f"Found course name: {selected_course_name}")
+            else:
+                 print(f"Course ID {course_id} not found in DB.")
+                 flash(f"Invalid course selected.", "warning")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)